src/Container.js

```diff
@@ -20,15 +20,30 @@ export default class Container extends React.Component {
       perRequest: 100,
       limitParam: 'perPage',
       params: getSASParams({
-        searchKey: 'name',
+        searchKey: 'name,alternateNames.name,description',
         filterKeys: {
+          agreementStatus: 'agreementStatus.value',
```

Comment:
The value here needs to be `agreementStatus.label`. Can you verify the same with all the other keys in this list.